virtcontainers/container_test.go

```diff
@@ -140,53 +134,37 @@ func TestContainerRemoveDrive(t *testing.T) {
 }
 
 func testSetupFakeRootfs(t *testing.T) (testRawFile, loopDev, mntDir string, err error) {
+	assert := assert.New(t)
 	if tc.NotValid(ktu.NeedRoot()) {
 		t.Skip(testDisabledAsNonRoot)
 	}
 
 	tmpDir, err := ioutil.TempDir("", "")
-	if err != nil {
-		t.Fatal(err)
-	}
+	assert.NoError(err)
 
 	testRawFile = filepath.Join(tmpDir, "raw.img")
-	if _, err := os.Stat(testRawFile); !os.IsNotExist(err) {
-		os.Remove(testRawFile)
-	}
+	_, err = os.Stat(testRawFile)
+	assert.True(os.IsNotExist(err))
```

Comment:
Is it better practice to fail the test if the file exists, instead of removing and continuing on like previously?